.github/workflows/jams.yml

```diff
@@ -0,0 +1,30 @@
+name: JSON Annotated Music Specification for Reproducible MIR Research
```

Comment:
Prefer to rename this file to `test.yml` or `ci.yml` - we'll likely want to add other workflows, and "jams" is not an informative name